### ISSUE TITLE
Latest postcss-nesting introduces is pseudo selector which aren't supported by old Chrome browsers

### ISSUE THREAD

**Issue by iliapir2** [ckeditor/ckeditor5#11730]
## 📝  Reproduction steps

1. Install latest release of CKEditor
2. Open the sample page in Chrome 85 
3. The editor toolbar layout is broken 

### ✔️ Expected result

The editor toolbar should be as before

### ❌ Actual result

 The editor toolbar layout is broken 

### ❓ Possible solution

I found that when you pass `noIsPseudoSelector: true` to  `postcss-nesting` plugin, it works as expected.
Source to patch:

https://github.com/ckeditor/ckeditor5-dev/blob/master/packages/ckeditor5-dev-utils/lib/styles/getpostcssconfig.js
Line: 26
Should be updated to:
```
	require('postcss-nesting')({
				noIsPseudoSelector: true
			}),
```

If you want I can provide you with PR to fix this issue.

## 📃 Other details

* Browser: Chrome 85
* OS: MacOs
* First affected CKEditor version: 34

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by BruTyler** [comment]
We've met the same problem... Chrome v83.
I'd like to have more complex options to customize interoperability for js/css

**Comment by BruTyler** [comment]
Additional problem with pseudo selector :not() - which is also available in Chrome 88+

**Comment by oleq** [comment]
Closed in ckeditor/ckeditor5-dev#768.
